src/shared/interfaces/index.ts

```diff
@@ -0,0 +1,6 @@
+import { SxProps } from "@mui/material";
```

Comment:
Maybe replace the folder's name with 'types' and create a file naming theme or style.
use the index file to expose all the diff. files in the types folder